README.md

```diff
@@ -244,4 +244,4 @@ registerKerasImageUDF("my_keras_inception_udf", InceptionV3(weights="imagenet"),
 ### Estimator
 
 ## Releases:
```

Comment:
Shall we keep the original `0.1.0` release note and add some description for the `0.2.0`release?
Thanks!